lib/N3Lexer.js

```diff
@@ -319,9 +319,10 @@ N3Lexer.prototype = {
       }
 
       // Emit the parsed token
-      callback(null, { line: line, type: type, value: value, prefix: prefix });
+      var token = { line: line, type: type, value: value, prefix: prefix };
+      callback(null, token);
       this._prevTokenType = type;
-
+      this._prevToken = token;
```

Comment:
did some eyeballing on the N3Lexer-perf script (ran on an ntriple file with 4million statements). Memory footprint is the same. User time _might_ have decreased, but with around 1 or 2% at most